x-pack/plugins/fleet/common/types/models/package_policy.ts

```diff
@@ -77,8 +77,17 @@ export interface PackagePolicy extends Omit<NewPackagePolicy, 'inputs'> {
   created_by: string;
 }
 
+// Used in list view tables where virtual/flattened fields are added
```

Comment:
nitpick but any reason to not keep this in public closer than where it's used?

Reply:
Nope can easily be moved